### PR TITLE
catalog

### PR DESCRIPTION
**Pull request by anastasia-radchencko**
https://anastasia-radchencko.github.io/layout_catalog/
https://anastasia-radchencko.github.io/layout_catalog/report/html_report/

### CODE REVIEW

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good, but pay attention to this point 
image
image

and it should look like this when you add a new item
image

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Nothing should be broken if I add more cards 
image

File: src/styles/normalize.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
```
Comment (stanislav-topikha) [inline src/styles/normalize.scss:6]: add fallback font

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    width: $card-width;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-top: 0;
+    }
+  }
```
Comment (stanislav-topikha) [inline src/styles/blocks/card.scss:114]: No external positioning for BEM blocks, event through modifiers, move to element and mix to card

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  flex-direction: column;
+  justify-content: center;
+
+  background-color: $standart-color;
+  border: 1px solid $card-border-color;
+  border-radius: $main-border-radius;
+
+  padding: 0 16px 16px;
+
+  &-container {
```
Comment (stanislav-topikha) [inline src/styles/blocks/card.scss:12]: what is this? BEM? 
image
why does it located outside of block?

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**